CMakeLists.txt

```diff
@@ -58,119 +60,162 @@ message(STATUS "This is a development version of Tasmanian")
 #   but only if Tasmanian_ENABLE_CUDA=OFF,
 #   Tasmanian_ENABLE_CUDA=ON absolutely requires find_package(CUDA)
 #
-# Optional to be added to the alternatives above
+# Optional that can be be added to the manual alternatives above
+#     -D OpenMP_CXX_LIBRARIES:STRING
 #     -D MPI_CXX_INCLUDE_PATH
 #     -D MPI_COMPILE_FLAGS
 #     -D MPI_LINK_FLAGS
 #     -D CUDA_INCLUDE_DIRS
 #     -D CUDA_LIBRARIES
 #     -D CUDA_cusparse_LIBRARY
+#     -D PYTHON_VERSION_MAJOR
+#     -D PYTHON_VERSION_MINOR
 #
 ########################################################################
-option(USE_XSDK_DEFAULTS "Enable xSDK compatibility" OFF)
+
+########################################################################
+# The build process follows these steps:
+# 1. Define all options and assign default values
+# 2. Perform sanity check
+#    - compliance with XSDK build policies
+#    - resolve conflicts, e.g., Python requires shared libs
+#    - call find_package() for options that need packages
+#    - fail or fallback if find_package() fails for some option
+# 3. Enable testing and set the master config file
+# 4. Add all subdirectories in the correct order
+#    - Interfaces require SparseGirds, but do not depend on each other
+#    - DREAM requires SparseGrids
+#    - SparseGrids does not require other sub-dirs
+# 5. Add the "make test_install" command
+# 6. Install top level files
+#    - cmake export file
+#    - master config file
+#    - CMakeLists.txt for the examples (include OpenMP hack if needed)
+#    - shell (bash) source file that defines paths
+# 7. Print final messages
+########################################################################
+
+option(USE_XSDK_DEFAULTS        "Enable xSDK compatibility"              OFF)
```

Comment:
Is this variable name imposed by XSDK?

Reply:
Yes: 

`https://xsdk.info/policies/`

check the second large link: 

`xSDK community installation policies: GNU Autoconf and CMake options [click here] – version 0.3.0, Nov 6, 2017`